File: day1/1.py
def fuel(mass):
    return (mass // 3) - 2

if __name__ == "__main__":
    with open("input.txt") as file:
        masses = file.readlines()
    masses = [int(x.strip()) for x in masses]
    print(sum(fuel(mass) for mass in masses))

File: day1/1.rs
use std::fs;

fn fuel(mass: i32) -> i32 {
    mass / 3 - 2
}

fn main() {
    let data = fs::read_to_string("input.txt")
        .expect("Something went wrong reading the file");
    let mut total_fuel = 0;
    for mass in data.lines() {
        total_fuel += fuel(mass.parse().unwrap());
    };
    print!("total_fuel is {}", total_fuel);
}